File: CS/NetFramework/net48/06_Display/ShowOverflowTip.cs
using System;

namespace InputManWin13_Demo._06_Display
{
    public partial class ShowOverflowTip : InputManWin13_Demo.DemoBase_settings
    {
        public ShowOverflowTip()
        {
            InitializeComponent();
            InitializeForm();
        }

        private void InitializeForm()
        {
            // ***** 初期設定
            checkBox1.CheckedChanged += new EventHandler(checkBox1_CheckedChanged);
            gcComboBox1.ListColumns[0].Width = gcComboBox1.Width;

            // ***** 初期値
            checkBox1.Checked = true;
            gcComboBox1.DropDown.AllowResize = false;
        }

        private void checkBox1_CheckedChanged(object sender, EventArgs e)
        {
            // ShowOverflowTipプロパティ
            gcTextBox1.ShowOverflowTip = checkBox1.Checked;
            gcComboBox1.ShowOverflowTip = checkBox1.Checked;
            gcListBox1.ShowOverflowTip = checkBox1.Checked;
        }
    }
}

File: VB/NetFramework/net48/06_Display/ShowOverflowTip.vb
Namespace _06_Display
    Partial Public Class ShowOverflowTip
        Inherits InputManWin13_Demo.DemoBase_settings
        Public Sub New()
            InitializeComponent()
            InitializeForm()
        End Sub

        Private Sub InitializeForm()
            ' ***** 初期設定
            gcComboBox1.ListColumns(0).Width = gcComboBox1.Width

            ' ***** 初期値
            checkBox1.Checked = True
            gcComboBox1.DropDown.AllowResize = False
        End Sub

        Private Sub checkBox1_CheckedChanged(ByVal sender As Object, ByVal e As EventArgs) Handles checkBox1.CheckedChanged
            ' ShowOverflowTipプロパティ
            gcTextBox1.ShowOverflowTip = checkBox1.Checked
            gcComboBox1.ShowOverflowTip = checkBox1.Checked
            gcListBox1.ShowOverflowTip = checkBox1.Checked
        End Sub
    End Class
End Namespace
